scripts/load_tester/start.py

```diff
@@ -0,0 +1,48 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+TX_SERVICE_URL = os.getenv("TRANSACTION_SERVICE_URL")
+PRINT_FORMAT = "{0:<10} {1:>8}::{2:>8}"
+
+
+def get_base_url() -> str:
+    if "staging" not in TX_SERVICE_URL:
+        return "https://safe-client.rinkeby.gnosis.io" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client.mainnet.gnosis.io"
+    else:
+        return "https://safe-client-rinkeby.staging.gnosisdev.com" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client-mainnet.staging.gnosisdev.com"
+
+
+def load_safes() -> list[str]:
+    print("Loading remote ...")
+    top_safes_url = TX_SERVICE_URL + "/api/v1/analytics/multisig-transactions/by-safe/?limit=300"
+    print(top_safes_url)
+    response = requests.get(top_safes_url)
+    return list(map(lambda safe: safe['safe'], response.json()['results']))
+
+
+base_gateway_url = get_base_url()
+safes = load_safes()
+print("Top 300 safes:")
+print("\n\t" + "\n\t".join(safes))
```

Comment:
Is it really helpful to pring all 300 safe addresses? I would then probably rather have 1 log statement when we start requesting the data (e.g. first line in the for loop)

Reply:
nah, I just thought it was nice. I will get rid of it